src/app.ts

```diff
@@ -73,9 +73,12 @@ app.post('/hook/v1/status', async (req, res, next) => {
     }
 });
 
-const cloudManager = new CloudManager({
-    cloud: 'aws',
-});
+const cloudManager = new CloudManager(
```

Comment:
This should be added to the CloudManagerOptions so that the oracleOptions field could have this instanceConfigurationId.